SparkPost/matchbox
# FE-844 Update Blue 700

### What Changed
- Updates blue 700 to `#1273E6` for accessibility requirements

**Extra:**
- Removes token compiled assets from git – `dist` `docs`
- Resolves tokens correctly in Storybook
- Runs prettier on `yml` token files

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify components that use that blue use `#1273E6` (ie. Check Button story)

### PR Checklist
N/A - Changes not applicable to < 4.0.0
